app/src/main/res/layout/activity_welcome.xml

```diff
@@ -37,12 +37,12 @@
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
         android:fontFamily="@font/museo_san_rounded700"
-        android:text="Best iranian VPN"
+        android:text="@string/best_iranian_vpn"
```

Comment:
Please change iranian name to something else!

Reply:
chaged to :
```
<string name="welcome_title">Best iranian VPN</string>
```